### DISCUSSION TITLE
Block/filter out by keyword or author.

### DISCUSSION THREAD

**Discussion by FifthDream** [BlafKing/sd-civitai-browser-plus#297] [Ideas]
LOVE the extension. Using it to browse new LORAs is part of my daily ritual. 

Would it be technically possible to have filters to block seeing models with specific keywords, or ones posted by specific creators? For example, you can block creators on the CivitAI site itself, and it would be great to be able to do it in the extension. (If that's already a feature and i somehow missed it, i apologize!)

Thanks so much for listening!

**Comment by atmogenic** [comment]
yes please, i'm surprised a content filter didn't come with this tool, there is a reason that the site has content filters and yet this didn't come with it. I really don't want to be seeing furry characters with massive c*cks on the thumbnails or some gross nsfw thumbnail. keywords, users and tags would do nicely
